docs/http_int_response.md

```diff
@@ -0,0 +1,13 @@
+# HTTP(S) Service
+
+
+Why is my `int` response a `string`?
+```
+$ curl -X POST "localhost:8013/flags/myIntFlag/resolve/int"
```

Comment:
It may be a good opportunity to refactor the example a bit. Making the code block a shell would allow users to copy the command in an easier way.

```sh
curl -X POST "localhost:8013/flags/myIntFlag/resolve/int"
```